plugins/zenoh-backend-traits/src/config.rs

```diff
@@ -76,6 +77,25 @@ impl Default for ReplicaConfig {
     }
 }
 
+// The configuration for periodic garbage collection of metadata in storage manager
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct GarbageCollectionConfig {
+    // The duration between two garbage collection events
+    // The garbage collection will be scheduled as a periodic event with this period
+    pub garbage_collection_period: Duration,
+    // The metadata older than this parameter will be garbage collected
+    pub garbage_collection_delay: Duration,
```

Comment:
Just call the field ?*age*?

Reply:
I renamed it as _lifespan_ which I believe is slightly better than _age_. Do you agree?

Reply:
I agree